src/journal/journald-kmsg.c

```diff
@@ -436,9 +437,9 @@ int server_open_kernel_seqnum(Server *s) {
                 return 0;
         }
 
-        r = posix_fallocate(fd, 0, sizeof(uint64_t));
+        r = posix_fallocate_loop(fd, 0, sizeof(uint64_t));
         if (r != 0) {
-                log_error_errno(r, "Failed to allocate sequential number file, ignoring: %m");
+                log_error_errno(-r, "Failed to allocate sequential number file, ignoring: %m");
```

Comment:
no need to negate log_error_errno() internally uses abs() anyway, so that you either pass a negative or a positive errno, it doesn#t matter